src/store/modules/order/actions.ts

```diff
@@ -100,6 +100,34 @@ const actions: ActionTree<OrderState, RootState> = {
       showToast(translate("Something went wrong"));
     }
     return resp;
+  },
+
+  async getPOHistory({ commit, state }, payload) {
+    let resp;
+
+    try {
+      const params = {
+        "inputFields":{
+          "orderId": [payload.order.orderId],
+          "orderId_op": "in"
+        },
+        "entityName": "ShipmentReceiptAndItem",
+        "noConditionFind": "Y"
+      }
```

Comment:
You could find specific fields
https://github.com/hotwax/job-manager/pull/14/files#diff-0b502070a993aa6e98c79c3fd77f07b8e85b2939b0c8e4408573fd72de0446d6R12